Fix the problem that zxing can't find the character encoding

```
Caused by: java.nio.charset.UnsupportedCharsetException: SJIS
	at java.base/java.nio.charset.Charset.forName(Unknown Source)
	at com.google.zxing.common.StringUtils.<clinit>(StringUtils.java:35)
	... 139 common frames omitted
```

--- a/composeApp/build.gradle.kts
+++ b/composeApp/build.gradle.kts
@@ -48,6 +48,7 @@ compose.desktop {
 
         nativeDistributions {
             targetFormats(TargetFormat.Dmg, TargetFormat.Msi, TargetFormat.Deb)
+            modules("jdk.charsets")
 
             packageName = "Clipevery"
             packageVersion = "1.0.0"

